install/linux/usr/share/odemis/sim/fastem-sim-asm.odm.yaml

```diff
@@ -309,7 +309,12 @@ FASTEM-sim: {
 "Optical Objective": {
     class: static.OpticalLens,
     role: lens,
-    init: {mag: 40}, # ratio, magnifying
+    init: {
+        # Adjust to have a small depthOfField, which makes the focus blur stronger.
```

Comment:
Would it be possible to add something like higher magnification is a stronger simulated blur?